_data/meltano/extractors/tap-db2/danielptv.yml

```
@@ -0,0 +1,72 @@
+capabilities:
+- about
+- catalog
+- discover
+- state
+- stream-maps
+description: "Tap-DB2 is a Singer tap for IBM DB2 data sources."
+domain_url: https://github.com/danielptv/tap-db2
+executable: tap-db2
+keywords:
+- meltano_sdk
+- ELT
+- IBM DB2
+- DB2
+label: DB2
+logo_url: /assets/logos/extractors/db2.png
+maintenance_status: active
+name: tap-db2
+namespace: tap_db2
+next_steps: ''
+pip_url: git+https://github.com/danielptv/tap-db2.git
+quality: unknown
```

```suggestion
quality: silver
```

@danielptv I think its safe to call this silver for now since you're using it and presumably will be responsive to issues/etc.